### PR TITLE
Minor JavaDoc fixes in IJsonRpcClient.java

### PR DESCRIPTION
**Pull request by msgilligan**
Fix minor errors:

`arguments` -> `argument`
`returnType` -> `<T>`

### CODE REVIEW

File: src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 	 * Invokes the given method with the given argument.
 	 *
 	 * @param methodName the name of the method to invoke
-	 * @param arguments the arguments to the method
+	 * @param argument the arguments to the method
```
Comment (vtintillier) [inline src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java]: You may want to change "the arguments to the method" to "the argument to the method" then

File: src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public Object invoke(String methodName, Object argument, Type returnType, Map<St
 	 *
 	 * @param methodName the name of the method to invoke
 	 * @param argument the arguments to the method
-	 * @param returnType the return type
```
Comment (vtintillier) [inline src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java]: should "@param clazz" be added also?

File: src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java (hunk 1 above)
Comment (msgilligan) [inline src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java]: Good point. I was just trying to make JavaDoc warnings/errors go away. I'll fix that too and push a new commit.

File: src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java (hunk 2 above)
Comment (msgilligan) [inline src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java]: Yeah. I fixed that and also a similar fix on the following method.  (I use IntelliJ to automatically generate the template JavaDoc comment and copy that  order of parameters -- it puts the type parameters last.)

File: src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java [hunk 3]
```diff
@@ -87,8 +87,8 @@ public Object invoke(String methodName, Object argument, Type returnType, Map<St
 	 * an object of the given type, or null if void.
 	 *
 	 * @param methodName the name of the method to invoke
-	 * @param arguments the arguments to the method
-	 * @param returnType the return type
+	 * @param argument the arguments to the method
+	 * @param <T> the return type
 	 * @param extraHeaders extra headers to add to the request
 	 * @return the return value
 	 * @throws Throwable on error
```
Comment (msgilligan) [inline src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java]: This one needs a `clazz` parameter, too.

File: src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java (hunk 2 above)
Comment (msgilligan) [inline src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java]: @vtintillier See: a1e4af03b2762ced4ca3c1d03a0b9d6c32ce7987

**Comment by gaborbernat** [comment]
@msgilligan can you fix the conflicts?

**Comment by vtintillier** [comment]
sorry for the conflicts, I thought this PR had been closed weeks ago already :-(

**Pull request closed without merging**